### DISCUSSION TITLE
Google Maps can't remember login state

### DISCUSSION THREAD

**Discussion by wzxu** [PlayCover/PlayCover#589] [Q&A]
So I installed Google Maps both from online resource and also my own dump from a jailbroken iPhone. They installed and ran beautifully. However it just can't seem to remember my login state.

At first, I though it only resets when I restart the app, but then I realize it resets even when I switch to another space in Mission Control and back, making it barely usable. 

Does this have something to do with the app login issue mentioned in the documentation? (I though it may refer to app showing error when you try to login, but I have no problem logging in, it just won't stick.) Should I disable SIP and try it?

Would appreciate if someone who also installed Google apps can share their experience, or explain a bit why SIP interferes with app login. Thanks!

**Comment by ZhichGaming** [comment]
This is not the app login issue mentioned in the documentation. App login issue is for when you cannot log in at all in an app, although it can still apply for other situations.

**Reply by wzxu** [reply]
Thanks for clarifying. I'll wait a bit to see if anyone else has similar experience. If not guess I don't have other choice than to give this a try.
